pkg/api/server/db/model.go

```diff
@@ -6,6 +6,7 @@ type Result struct {
 	Parent string `gorm:"primaryKey;index:results_by_name,priority:1"`
 	ID     string `gorm:"primaryKey"`
 	Name   string `gorm:"index:results_by_name,priority:2"`
+	Data   []byte
```

Comment:
I think we might want to inline these fields directly, so that results can be sorted / indexed by the timestamp values. Unlike records, results don't contain arbitrary user blobs, so it doesn't make as much sense to store the API proto here.

Reply:
Yeah exactly, I'm working on this